### DISCUSSION TITLE
Spatial map tab is not supported for when using `select` RQL

### DISCUSSION THREAD

**Discussion by kamranayub** [ravendb/ravendb#16867] [General]
This may be more of a bug if it's confirmed. I have a Map-Reduce index that outputs results to a collection (`RegionalProducts`). 

When I issue a collection spatial query, the tab to show the Spatial Map does not show up anymore:

![image](https://github.com/ravendb/ravendb/assets/563819/be1194b1-5332-4805-b903-d5f8fddfc5e3)

Do you know if this is a known issue? Also, it would be awesome to get that tab on index queries, too 😄 

**Edit:** Updated to reflect discussion below -- looks like it's the `select` statement causing the issue.

**Comment by ppekrol** [comment]
Could be related: https://issues.hibernatingrhinos.com/issue/RavenDB-20620/Spatial-map-view-does-Not-show-for-dynamic-query-with-order-by
Do you also see any errors in the browser console log?

**Reply by kamranayub** [reply]
Oh, interesting, it looks like it works WITHOUT the `select` statement. The `order by` seems OK.

![image](https://github.com/ravendb/ravendb/assets/563819/1ec867b2-79d0-4009-b2c1-86b34c70484c)

**Reply by kamranayub** [reply]
So, at least this unblocks me, I don't need the select -- but it was unexpected so not sure if that's still a bug...

**Reply by ayende** [reply]
The spatial information is removed by the select, most likely.
Note sure whatever this is a bug or not
